res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_001/test.rule

```diff
@@ -0,0 +1,14 @@
+# To check mixed-precision quantization.
+# Conv, Mul: int16, Add: u8
+# Quantize Ops are inserted before Conv and after Mul.
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "CONV_INT16"              $(tensor_dtype ofm_conv) '=' INT16
+RULE    "WEIGHTS_INT16"           $(tensor_dtype filter) '=' INT16
+RULE    "BIAS_INT32"              $(tensor_dtype bias) '=' INT64
```

Comment:
optional) `BIAS_INT32` has `INT32` but result is `INT64`. maybe match to `BIAS_INT64`?

Reply:
Oops.. Thanks for finding!